nntrainer/layers/rnn.h

```diff
@@ -93,10 +93,13 @@ class RNNLayer : public Layer {
   void setProperty(const PropertyType type,
                    const std::string &value = "") override;
 
+  void setActivation(ActivationType acti_type) override;
+
   static const std::string type;
 
 private:
   unsigned int unit;
+  ActiFunc acti_func;
```

Comment:
Please set the default activation function to `TanH`. RNN defaults to TanH activation and a user specifying RNN without activation must still remain a valid RNN.

Reply:
come up this next PR

Reply:
Great.